IFTPipeline.jl/src/landmask.jl

```diff
@@ -24,3 +24,33 @@ function landmask(; input::String, output::String, landmask_fname::String="landm
     @info "Landmask created and serialized succesfully."
     return nothing
 end
+
+
+"""
+    landmask_single(; input, output_non_dilated, output_dilated)
```

Comment:
What is the rationale for the `x_single` naming convention?

Reply:
Because they are operating on single files rather than batches. I'm open to renaming them, but it's tricky because they share signatures with the batch versions. 

Option 0:
We could include some other method to differentiate between them – a sentinel value you include to switch the mode from batch to single file.

Option 1:
- rename the functions to `landmask`, `extractfeatures` and use
- `src/batch-cli.jl` and `src/single-cli.jl` as the entry points, splitting them up, and 
- provide two different docker containers.

Option 2: 
- remove the batch versions and just leave the single file variants.

Reply:
> Option 0:
We could include some other method to differentiate between them – a sentinel value you include to switch the mode from batch to single file.

This will not be good as it would make the function not type stable which is bad in Julia. Now that I understand the reasoning for the naming, I am more okay with it. Thanks.